Src/Managers/font_manager/font_manager.py

```
@@ -0,0 +1,90 @@
+                            font_name, font_config['sizes'][0], font_ids[0])
+            self.fonts[font_name][font_config['sizes'][0]] = font
+
+            for size, curr_id, next_id in zip(font_config['sizes'][1:], font_ids[:-1], font_ids[1:]):
+
+                font: FontUnit = get_userdata(curr_id)
+                next_font = FontUnit(Path(font_config['path']).resolve(), font_config['hints'],
+                                     font_name, size, next_id, prev = font)
+                font.next = next_font
+                self.fonts[font_name][size] = font
+
+            if not font_config.get('default'): continue
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)